bin/README.md

```diff
@@ -0,0 +1,2 @@
+# bin
+Where the yt-dlp binary is stored by KMQ.
```

Comment:
Why do we need a README for this? Also, if we have it, shouldn't the text be more generic